mcp-server/test_mcp_mocked.py

```
@@ -0,0 +1,243 @@
+                )
+                
+                # Create and return the response
+                response = ContextResponse(
+                    context_items=context_items,
+                    metadata=response_metadata
+                )
+                
+                return response
+                
+            except ValueError as e:
+                raise ValueError(str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)